src/converters/lintConfigs/rules/ruleConverters/align.ts

```diff
@@ -29,6 +29,10 @@ export const convertAlign: RuleConverter = (tslintRule) => {
 
     return {
         rules: [
+            {
+                ruleName: "indent",
+                ruleSeverity: "off",
+            },
```

Comment:
Based on #149, I think it is safe to  update the [Typescript ESLint Roadmap](https://github.com/typescript-eslint/typescript-eslint/blob/61e035356ced5186040eb4aeb78bd74c1c1ce4b8/packages/eslint-plugin/ROADMAP.md) with this conversion. @JoshuaKGoldberg